specification/servicefabric/resource-manager/Microsoft.ServiceFabric/2016-09-01/servicefabric.json

```diff
@@ -314,10 +314,10 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/environments/{environment}/clusterVersions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/clusterVersions": {
       "get": {
         "tags": [
-          "ClusterVersionsListOperation"
+          "ClusterVersion"
```

Comment:
Should this be in the same `2016-09-01` version? If the previous version was technically correct we should update the entire spec in a new version